src/deploy/functions/prepareFunctionsUpload.ts

```diff
@@ -89,6 +91,16 @@ async function packageSource(
         mode: file.mode,
       });
     }
+    for (const name of additionalSources) {
+      const absPath = path.join(projectDir, name);
+      const mode = fs.statSync(absPath).mode;
```

Comment:
what happens if this file doesn't exist?

Reply:
Added error handling.